ui/raidboss/data/06-ew/dungeon/stigma_dreamscape.txt

```diff
@@ -0,0 +1,125 @@
+1028.4 "Auto-mobile Assault Cannon" sync / 1[56]:[^:]*:Arch-Lambda:63AB:/
+1028.7 "Tread x4" duration 4 #sync / 1[56]:[^:]*:Arch-Lambda:63AC:/
+1032.1 "--sync--" sync / 1[56]:[^:]*:Arch-Lambda:63AD:/
+1036.3 "Wave Cannon" sync / 1[56]:[^:]*:Arch-Lambda:63AF:/ window 30,30
+1049.5 "Atomic Flame" sync / 1[56]:[^:]*:Arch-Lambda:63B4:/
+1061.8 "--sync--" sync / 1[56]:[^:]*:Arch-Lambda:63AA:/
+1070.7 "Auto-mobile Sniper Cannon" sync / 1[56]:[^:]*:Arch-Lambda:63B0:/
+1071.2 "Entrench" sync / 1[56]:[^:]*:Arch-Lambda:63B1:/
+
+1080.9 "Wheel" sync / 1[56]:[^:]*:Arch-Lambda:63B5:/ window 30,30
+1091.1 "--sync--" sync / 1[56]:[^:]*:Arch-Lambda:63AA:/
+1098.6 "Auto-mobile Assault Cannon" sync / 1[56]:[^:]*:Arch-Lambda:63AB:/
+1098.9 "Tread x4" duration 4 #sync / 1[56]:[^:]*:Arch-Lambda:63AC:/
+1102.2 "--sync--" sync / 1[56]:[^:]*:Arch-Lambda:63AD:/
+1106.4 "Wave Cannon" sync / 1[56]:[^:]*:Arch-Lambda:63AF:/ window 30,30
+1119.6 "Atomic Flame" sync / 1[56]:[^:]*:Arch-Lambda:63B4:/
+1131.9 "--sync--" sync / 1[56]:[^:]*:Arch-Lambda:63AA:/
+1140.8 "Auto-mobile Sniper Cannon" sync / 1[56]:[^:]*:Arch-Lambda:63B0:/
+1141.3 "Entrench" sync / 1[56]:[^:]*:Arch-Lambda:63B1:/
+
+# TODO: Probably the loop point? More data needed.
+1151.0 "Wheel" sync / 1[56]:[^:]*:Arch-Lambda:63B5:/ window 30,30 jump 1080.9
+1168.7 "Auto-mobile Assault Cannon"
+1169.0 "Tread x4"
+1176.5 "Wave Cannon"
+1189.7 "Atomic Flame"
+1210.9 "Auto-mobile Sniper Cannon"
+1211.4 "Entrench"
+
+
+#~~~~~~~~~~#
+# STIGMA-4 #
+#~~~~~~~~~~#
+
+# -ii 6435
+
+2000.0 "--sync--" sync / 00:0839::A-4 Headquarters will be sealed off/ window 2000,1
+2010.0 "AI Takeover" sync / 1[56]:[^:]*:Stigma-4:6429:/
+2015.6 "Touchdown?" sync / 1[56]:[^:]*:Hybrid Dragon:68F9:/
+2024.7 "Proto-wave Cannon?" sync / 1[56]:[^:]*:Omega Frame:(642A|642B):/
+2024.7 "Fire Breath?" sync / 1[56]:[^:]*:Hybrid Dragon:642E:/
+2031.3 "AI Takeover" sync / 1[56]:[^:]*:Stigma-4:6429:/
+2037.1 "Touchdown?" sync / 1[56]:[^:]*:Hybrid Dragon:68F9:/
+2044.5 "Self-Destruct?" # 14.7s cast time, cast seems to start 5.1s after Breath/Cannon
+2046.2 "Fire Breath?" sync / 1[56]:[^:]*:Hybrid Dragon:642E:/
+2046.2 "Proto-wave Cannon?" sync / 1[56]:[^:]*:Omega Frame:(642A|642B):/
+2052.6 "AI Takeover" sync / 1[56]:[^:]*:Stigma-4:6429:/
+2061.5 "Rush" sync / 1[56]:[^:]*:Proto-rocket Punch:642D:/ window 30,30
+2066.0 "Self-Destruct?"
+2067.0 "--sync--" sync / 1[56]:[^:]*:Stigma-4:6433:/
+2077.0 "Electromagnetic Release" sync / 1[56]:[^:]*:Stigma-4:6434:/
+2085.1 "--sync--" sync / 1[56]:[^:]*:Stigma-4:6431:/
+2095.1 "Electromagnetic Release" sync / 1[56]:[^:]*:Stigma-4:6432:/
+2108.2 "Mindhack" sync / 1[56]:[^:]*:Stigma-4:6430:/ window 30,30
+2113.4 "--sync--" sync / 1[56]:[^:]*:Stigma-4:(6431|6433):/
+2123.4 "Electromagnetic Release" sync / 1[56]:[^:]*:Stigma-4:(6432|6434):/
+2132.5 "Atomic Ray" sync / 1[56]:[^:]*:Stigma-4:6436:/
+2142.7 "Multi-AI Takeover" sync / 1[56]:[^:]*:Stigma-4:6C4B:/
+2148.4 "Touchdown?" sync / 1[56]:[^:]*:Hybrid Dragon:68F9:/
```

Comment:
Why does touchdown get a question mark? (and firebreath above too?)

Reply:
Because each AI Takeover is either a Hybrid Dragon spawn *or* an Omega Frame spawn, and their abilities are different, but the timings on everything else are otherwise the same. They don’t alternate either, although I don’t *think* I’ve seen a run with all Dragons or Frames.